src/runner/runtime.ts

```diff
@@ -34,6 +34,7 @@ export class RunnerRuntime extends createEventEmitter<SpectralEvents>() {
   public spawn(): Pick<RunnerRuntime, 'on'> {
     return this.persist(
       Object.freeze({
+        // eslint-disable-next-line @typescript-eslint/unbound-method
         on: this.hijackDisposable(this.on),
```

Comment:
On my machine, eslint was throwing an `@typescript-eslint/unbound-method` error. So I added this, since I assume you know what you're doing and are being intentional.